lib/usd/ui/layerEditor/sessionState.cpp

```diff
@@ -23,18 +23,19 @@ void SessionState::setAutoHideSessionLayer(bool hideIt)
     Q_EMIT autoHideSessionLayerSignal(_autoHideSessionLayer);
 }
 
-void SessionState::setStage(PXR_NS::UsdStageRefPtr const& in_stage)
+void SessionState::setStageEntry(StageEntry const& in_entry)
```

Comment:
I would like this method to continue to be named setStage instead of set"StageEntry"

Reply:
I guess its a style thing, but I think keeping the same name is confusing.

Reply:
it's setting the current stage, the StageEntry is just the 'how'. Like previously, it wasn't called "setStageRefPtr"

Reply:
Yea but its not just setting the current stage, its setting the current stage, the proxy shape id the path and the display name, so calling it setStage implies its only doing one of those things.